ablock.go

```diff
@@ -633,10 +632,19 @@ func GetABlock(keymr string) (ablock *ABlock, raw []byte, err error) {
 	return wrap.ABlock, raw, nil
 }
 
-// GetABlockByHeight requests an ABlock of a specific height from the factomd
-// API.
-func GetABlockByHeight(height int64) (ablock *ABlock, raw []byte, err error) {
-	params := heightRequest{Height: height}
+// GetABlock requests a specific ABlock from the factomd API with the raw data
+func GetABlock(keymr string) (ablock *ABlock, raw []byte, err error) {
+	return getABlock(keymr, false)
+}
+
+// GetSimpleABlock requests a specific ABlock from the factomd API without the raw data
+func GetSimpleABlock(keymr string) (ablock *ABlock, err error) {
+	ablock, _, err = getABlock(keymr, true)
+	return
+}
+
+func getABlockByHeight(height int64, noraw bool) (ablock *ABlock, raw []byte, err error) {
+	params := heightRequest{Height: height, NoRaw: noraw}
```

Comment:
What do you think of even include that factomd task in Wax immedaiately? The ratio benefit/cost looks pretty good, should be easy change in factomd and we can finally get rid of those length raw data

Reply:
> Do we have a task (issue) on the factomd side to introduce the new param? If you confirm we don't can you create it and link the issues? Thanks!

https://github.com/FactomProject/factomd/pull/996



> What do you think of even include that factomd task in Wax immedaiately?

Depends on whether or not we have a feature freeze for Wax. If there's no freeze, I agree it can be included immediately.

Reply:
Yeah I was suggesting to get soon a feature freeze but I was still grooming through the issues and PR, and I think that one is pretty nice to have. I have added the PR you pointed to to the Wax project board